src/main.py

```diff
@@ -131,6 +131,10 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
             print(f"Task {i+1} / {len(test_tasks)}: Environment failed "
                   f"with error: {e}")
             continue
+        except ApproachFailure as e:
+            print(f"Task {i+1} / {len(test_tasks)}: Approach failed "
+                  f"with error: {e}")
```

Comment:
can you say "Approach failed at policy execution time with error:" to distinguish from above?

also, except (ApproachTimeout, ApproachFailure)

Reply:
@ronuchit re the second thing, what would `ApproachTimeout` mean at execution time?

Reply:
approach failed to compute the next action within the time limit, imagine it's like an online planner for example

Reply:
think of it not as execution time, but as policy inference time (same thing but slightly diff perspective)

Reply:
got it, makes sense